commit 5071ef936bf566bb554d59248cbac8fe9aafd1e8 (refs/changes/76/3476/3)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-02-27T23:30:19+00:00 (1 year, 7 months ago)
    
    TOOLS-1983 How to avoid hard failure of most builds on Feb 22 from GitHub weak crypto removal
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index 452837a..f4ee194 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1249,10 +1249,10 @@ cat <<EOF
     "build_platform": "20151126T062538Z",
     "image_name": "manta-storage",
     "image_description": "Manta storage",
-    "image_uuid": "fd2cc906-8938-11e3-beab-4359c665ac99",
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-      "pcre-8.21",
-      "zookeeper-client-3.4.3"
+      "pcre-8.38"
     ],
     "repos": [
       {"url": "git@github.com:joyent/manta-mako.git"}
